src/internal/group_api/requests.rs

```diff
@@ -94,6 +96,39 @@ impl TryFrom<GroupGetApiResponse> for GroupGetResult {
     }
 }
 
+#[derive(Deserialize, Debug, Clone, PartialEq)]
+#[serde(rename_all = "camelCase")]
+pub struct GroupCreateApiResponse {
+    pub(crate) id: GroupId,
```

Comment:
I have been making these response objects `pub(in crate:internal)` recently as we should never need these outside of the `internal` portion of the crate.